### ISSUE TITLE
Linux and macOS version of TaskExplorer

### ISSUE THREAD

**Issue by theuserbl** [DavidXanatos/TaskExplorer#39]
Because TaskExplorer using Qt as frontend, it could be possible to use the same frontend with different backends.

For Linux existing for example 
https://github.com/KrispyCamel4u/SysMonTask
where the backend can be used like the backend of ProcessHacker is used for the TaskExplorer for Windows.

So it could be the first taskmanager like program, which exists on more then one platform.

**Comment by DavidXanatos** [comment]
Yes it would be and if you check out the source there are already provisions for other platforms even an linux folder with almost nothign in it. it juts need work and time to add linux suport
